js/script.js

```diff
@@ -34,4 +33,142 @@ open.addEventListener('click', () => {
+    tags: ['html', 'Ruby on rails', 'css', 'javascript', 'github', 'Bootstraps'],
+    image: 'images/d4.png',
+    liveButton: 'live button',
+    sourceButton: 'See Source',
+    live: 'images/icon.png',
+    source: 'images/Icon-GitHub.png',
+  },
+];
+
+function display(index) {
+  const main = document.querySelector('main');
+  const {
+    id,
+    title,
+    devs,
+    description,
+    tags,
+    image,
+    liveButton,
+    sourceButton,
+    live,
+    source,
+  } = projects[index];
+  const dev1 = devs[0];
+  const dev2 = devs[1];
+  const dev3 = devs[2];
+  const dev4 = devs[3];
+  const dev5 = devs[4];
+  const tag1 = tags[0];
+  const tag2 = tags[1];
+  const tag3 = tags[2];
+  const tag4 = tags[3];
+  const tag5 = tags[4];
+  const tag6 = tags[5];
+  const projectPopup = document.createElement('div');
+  projectPopup.className = 'project-popup';
+  projectPopup.innerHTML = `
+     <div class="desktop-window">
+     <div class="all-desktop-popup"
+       <div class="desktop-top-popup">
+         <h2>${title}</h2>
+         <button type='button' class="X-button">X</button>
+       </div>
+       <ul class="devls"> 
+         <li class="dev1">${dev1}</li> 
+         <li class="dev2">${dev2}</li>
+         <li class="dev3">${dev3}</li>
+         <li class="dev4">${dev4}</li>
+         <li class="dev5">${dev5}</li>
+       </ul>
+       <img class="desktop-popup-snapshot image-${id}" src=${image} alt="project 1 snapshot">  
+       <div class="popup-footer">
+       <p class="p-popup">${description}</p> 
+       <section class=lang-button>
+        <ul class="prog-languages">  
+        <ul class="ul">
+         <li class="lang">${tag1}</li>   
+         <li class="lang">${tag2}</li>  
+         <li class="lang">${tag3}</li> 
+         </ul>
+         <ul class="ul1">
+         <li class="lang">${tag4}</li>   
+         <li class="lang">${tag5}</li>   
+         <li class="lang">${tag6}</li>  
+         </ul>
+       </ul>
+       <div class ="desktopup-button">  
+       <button type='button' class="popup-button">
+         ${liveButton}
+         <img class="logo-popup-button" src=${live}> 
+       </button>
+       <button type='button' class="popup-button">  
+         ${sourceButton}
+         <img class="logo-popup-button" src=${source}>  
+       </button>
+       </section>
+     </div>
+     </div>
+     </div>
+     </div>`;
+  main.appendChild(projectPopup);
```

Comment:
**[REQUIRED CHANGES]**

- Hey, nice job with the pop-up window however there is some issue with the design. Please compare your design with the Figma design to implement the changer below. check the video to see your pop up in action:

**your pop up**


https://user-images.githubusercontent.com/59397159/169666394-8af4ff7c-3c4b-47b2-924f-2aa50b14c27f.mp4

**Figma pop up desktop**

![figpop](https://user-images.githubusercontent.com/59397159/169666450-c21009b9-8d80-4aa7-ab81-99e56f9c0829.png)


**figma pop up mobile**

![mobpop](https://user-images.githubusercontent.com/59397159/169666514-230065b9-cedb-4c97-90b6-6b0fce70a4f4.png)

> - Please, adjust your style to match the design..

Reply:
**[REQUIRED CHANGES]**

- Please, kindly adjust the styling to your nav bar to match the figma design.


**figma**
![nav-fig](https://user-images.githubusercontent.com/59397159/169666608-f64d2df1-cdc7-408d-b877-9e162926235b.png)


**your design**
![nav](https://user-images.githubusercontent.com/59397159/169666611-c7dc165a-dbea-47a1-acec-302b58f69163.png)